static/includes/TNDataCollection.md

```diff
@@ -1,12 +1,13 @@
 &NewLine;
 
-When disabled, no information about system configuration and usage is collected.
-The system capacity and software version is still collected.
-
 The protocol for system data collection uses the same TCP ports as HTTPS (*443*) and passes through most firewalls as an outgoing web connection.
 If a firewall blocks the data collection or the data collection is disabled, there is no adverse impact to the TrueNAS system.
 
-Non-sensitive system data is used to identify the quality and operational trends in the fleet of TrueNAS systems used by the entire community.
+When **Usage collection** is disabled, anonymous usage statistics consisting only of the software version and total system capacity (e.g. TrueNAS SCALE 24.04.0, 55 TB) are still collected.
+No information about system configuration and usage is collected.
+
+When enabled, non-sensitive system data is transmitted to iXsystems.
+This data is used to identify the quality and operational trends in the fleet of TrueNAS systems used by the entire community.
 The collected data helps iXsystems identify issues, plan for new features, and determine where to invest resources for future software enhancements.
```

Comment:
Use of "when" seems repetitive but I understand why it is used. Also I think we can combine the two sentences into one. Maybe this as an alternative?
```suggestion
iXsystems collects this data and uses it to identify the quality and operational trends in the fleet of TrueNAS systems used by the entire community, and identify issues, plan for new features, and determine where to invest resources for future software enhancements.
```